tests/conformance/retry_strategy_test_data.json

```diff
@@ -2,247 +2,84 @@
-              "NOTIFICATION"
-            ]
-          },
-          {
-            "name": "storage.notifications.list",
-            "resources": [
-              "BUCKET",
-              "NOTIFICATION"
-            ]
-          },
-          {
-            "name": "storage.object_acl.get",
-            "resources": [
-              "BUCKET",
-              "OBJECT"
-            ]
-          },
-          {
-            "name": "storage.object_acl.list",
-            "resources": [
-              "BUCKET",
-              "OBJECT"
-            ]
-          },
-          {
-            "name": "storage.objects.get",
-            "resources": [
-              "BUCKET",
-              "OBJECT"
-            ]
-          },
-          {
-            "name": "storage.objects.list",
-            "resources": [
-              "BUCKET",
-              "OBJECT"
-            ]
-          },
-          {
-            "name": "storage.serviceaccount.get",
-            "resources": []
-          }
+          {"name": "storage.bucket_acl.get",                "resources": ["BUCKET"]},
+          {"name": "storage.bucket_acl.list",               "resources": ["BUCKET"]},
+          {"name": "storage.buckets.delete",                "resources": ["BUCKET"]},
+          {"name": "storage.buckets.get",                   "resources": ["BUCKET"]},
+          {"name": "storage.buckets.getIamPolicy",          "resources": ["BUCKET"]},
+          {"name": "storage.buckets.insert",                "resources": []},
+          {"name": "storage.buckets.list",                  "resources": ["BUCKET"]},
+          {"name": "storage.buckets.lockRetentionPolicy",   "resources": ["BUCKET"]},
+          {"name": "storage.buckets.testIamPermissions",    "resources": ["BUCKET"]},
+          {"name": "storage.default_object_acl.get",        "resources": ["BUCKET"]},
+          {"name": "storage.default_object_acl.list",       "resources": ["BUCKET"]},
+          {"name": "storage.hmacKey.delete",                "resources": ["HMAC_KEY"]},
+          {"name": "storage.hmacKey.get",                   "resources": ["HMAC_KEY"]},
+          {"name": "storage.hmacKey.list",                  "resources": ["HMAC_KEY"]},
+          {"name": "storage.notifications.delete",          "resources": ["BUCKET", "NOTIFICATION"]},
+          {"name": "storage.notifications.get",             "resources": ["BUCKET", "NOTIFICATION"]},
+          {"name": "storage.notifications.list",            "resources": ["BUCKET", "NOTIFICATION"]},
+          {"name": "storage.object_acl.get",                "resources": ["BUCKET", "OBJECT"]},
+          {"name": "storage.object_acl.list",               "resources": ["BUCKET", "OBJECT"]},
+          {"name": "storage.objects.get",                   "resources": ["BUCKET", "OBJECT"]},
+          {"name": "storage.objects.list",                  "resources": ["BUCKET", "OBJECT"]},
+          {"name": "storage.serviceaccount.get",            "resources": []}
```

Comment:
I must say this doesn't feel more readable / natural to my eyes.  Is there a JSON-linter you are applying here?  Or is this file being copied in from another source which applies it?

Reply:
The json file *will be* copied from another source in [conformance-tests/retry_tests.json](https://github.com/googleapis/conformance-tests/blob/master/storage/v1/retry_tests.json) once the test suites are finalized. Either formatting works for me. I chose to change the formatting mainly to (1) align with the source json file (2) seems less verbose and more readable to my eyes (--> easier to track all methods and resources in one view)

Reply:
@tseaver Let me know if there are further concerns. Thanks!